interp/interpreter.go

```diff
@@ -369,6 +369,11 @@ func (r *runner) run(fn *function, params []value, parentMem *memoryView, indent
 				nBytes := uint32(operands[3].Uint())
 				dstObj := mem.getWritable(dst.index())
 				dstBuf := dstObj.buffer.asRawValue(r)
+				if mem.get(src.index()).buffer == nil {
+					// Looks like the source buffer is not defined.
+					// This can happen with //extern or //go:embed.
+					return nil, mem, r.errorAt(inst, errUnsupportedRuntimeInst)
```

Comment:
for those watching at home, this error is recoverable (interp.isRecoverableError). returning this error doesn't fail, it just defers evaluation until runtime (this line is executed compiling the code).

Reply:
thanks for the explanation!